lib/matplotlib/artist.py

```diff
@@ -1421,6 +1421,10 @@ def setp(obj, *args, **kwargs):
       >>> setp(lines, 'linewidth', 2, 'color', 'r')  # MATLAB style
       >>> setp(lines, linewidth=2, color='r')        # python style
     """
+    
+    if ((isinstance(obj, np.ndarray) and not obj.size) or
```

Comment:
Wouldn't `if not objs` return false if `objs` is a list of empty lists? Or do you mean to check for `if not obj`?